src/Sarif.Driver.UnitTests/AnalyzeCommandTests.cs

```diff
@@ -30,12 +30,12 @@ public class AnalyzeCommandBaseTests
 
             var command = new TestAnalyzeCommand();
 
-            Assembly[] plugInAssemblies;
+            Assembly[] plugInAssemblies = null;
 
-            if (analyzeOptions.PlugInFilePaths != null)
+            if (analyzeOptions.DefaultPlugInFilePaths != null)
```

Comment:
Test code was hijacking the plug-in paths in order to hydrate the default plug in assemblies data. Since we now validate the plug in file paths, this is no longer acceptable and we have an explicit test options for this scenario.